In [2]:
import numpy as np

# Definición de Fuerza y Masa

## Cantidad de movimiento

La cantidad de movimiento, momento lineal, ímpetu, momentum o simplemente momento,​ es una magnitud física derivada de tipo vectorial que describe el movimiento de un cuerpo en cualquier teoría mecánica. En mecánica clásica, la cantidad de movimiento se define como el producto de la masa del cuerpo y su velocidad en un instante determinado.

$$
\vec{p} = m \cdot \vec{v} ~~~~ \vec{p} = m \frac{d \vec{r}}{d t}
$$

- $ p $: Cantidad de movimiento.
- $ m $: Masa.
- $ v $: Velocidad del movimiento.
- $ r $: desplazamiento en el movimiento.
- $ t $: tiempo.

Conservación de la cantidad de movimiento

$$
\sum_{i=1}^{n} \vec{p_{1i}} = \sum_{i=1}^{n} \vec{p_{0i}}
$$

## Fuerza

En física clásica, la fuerza (abreviatura F) es un fenómeno que modifica el movimiento de un cuerpo (lo acelera, frena, cambia el sentido, etc.) o bien lo deforma. Las fuerzas pueden representarse mediante vectores, ya que poseen magnitud y dirección.

$$
\vec{F} = \frac{d \vec{p}}{d t} ~~~~ \vec{F} = m \cdot \vec{a}
$$

- $ F $: Fuerza.
- $ p $: Cantidad de movimiento.
- $ a $: aceleración.
- $ t $: tiempo.

## Primera ley de newton

Si la fuerza resultante que actúa sobre una partícula es cero, la partícula permanecerá en reposo o se moverá con velocidad constante en línea recta.

$$
\sum_{i=0}^{n} \vec{F_i} = 0
$$

## Segunda ley de newton

Si la fuerza resultante que actúa sobre una partícula no es cero, la partícula tendrá una aceleración proporcional a la magnitud de la resultante y en la dirección de ésta.

$$
\sum_{i=0}^{n} \vec{F_i} = m \cdot \vec{a}
$$

## Tercera ley de newton

Las fuerzas de acción y reacción de cuerpos en contacto tienen la misma magnitud, la misma línea de acción y sentidos opuestos.

$$
\vec{F_1} = - \vec{F_2}
$$

## Fuerza de rozamiento

La fuerza de rozamiento o de fricción ($ F_r =  $) es una fuerza que surge por el contacto de dos cuerpos y se opone al movimiento.

### En reposo

La fuerza de rozamiento tiene el mismo módulo, dirección y sentido contrario de la fuerza horizontal (si existe) que intenta ponerlo en movimiento sin conseguirlo.

### En movimiento

Como la fuerza de rozamiento depende de los materiales y de la fuerza que ejerce uno sobre el otro.

$$
F_r = \mu \cdot F_n
$$

- $ F_r $: Fuerza de rozamiento.
- $ \mu $: Coeficiente de rozamiento.
- $ F_n $: Fuerza normal.

## Peso

En física moderna, el peso es una medida de la fuerza gravitatoria que actúa sobre un objeto.​ El peso equivale a la fuerza que ejerce un cuerpo sobre un punto de apoyo, originada por la acción del campo gravitatorio local sobre la masa del cuerpo. 

$$
W = m \cdot g
$$

- $ W $: Peso.
- $ m $: masa.
- $ g $: gravedad ($ g = 10 [m / s^2] $)

# Ejercicios

## Ejercicio 1

Encontrar la fuerza ($ F $) para que el objeto se mueva con velocidad constante hacia la derecha, el coeficiente de rozamiento del plano inclinado es $ \mu = 0.15 $.

![ejr 1](./img/ejr_1.png)

### Solución

Se gráfica las fuerzas que se involucran en el movimiento.

![ejr 1 sol](./img/ejr_1_2.png)

Primero se descomponen cada una de las fuerzas las fuerzas 

$$
25° = 0.43633 [rad]
\vec{W} = m \cdot g \cdot \vec{e}
\\
\vec{W}  = 12 [Kg] \cdot 10 [m/s^2] \cdot ( \cos \theta \vec{i} + \sin \theta \vec{j})
\\
\vec{W}  = 120 [Kg m/s^2] \cdot ( \cos 3.5779 \vec{i} + \sin 3.5779 \vec{j})
$$

In [33]:
theta = 2 * np.pi - 0.43633 - (np.pi / 2)
e = np.array([np.cos(theta), np.sin(theta)])
W = 120 * e
F_n = np.array([0, - W[1]])
print('W = ', W)

W =  [ -50.71393985 -108.75705175]


$$
\vec{W} = - 50.714 \vec{i} - 108.757 \vec{j} [N]
$$

Decimos que $ F_n = 108.757 [N] $, entonces F_r es:

$$
F_r = \mu \cdot F_n
\\
F_r = 0.15 \cdot 108.757 [N]
$$

In [31]:
mu = 0.15
F_r = np.array([mu * W[1], 0])
print('F_r = ', F_r)

F_r =  [-16.31355776   0.        ]


$$
F_r = 16.3136 [N] => \vec{F_r} = - 16.3136 \vec{i} [N]
$$

Una ves obtenido $ F_r $ se procede a realizar la sumatoria de fuerzas.

$$
\sum_ F = 0
\\
\vec{F} + \vec{F_n} + \vec{F_r} + \vec{W} = 0
\\
\vec{F} + 108.757 \vec{j} [N] - 16.3136 \vec{i} [N] - 50.714 \vec{i} - 108.757 \vec{j} [N] = 0
\\
\vec{F} = - \vec{F_n} - \vec{F_r} - \vec{W}
\\
\vec{F} = - 108.757 \vec{j} [N] + 16.3136 \vec{i} [N] + (50.714 \vec{i} + 108.757 \vec{j}) [N]
$$

In [34]:
F = - F_n - F_r - W
print('F = ',F)

F =  [67.02749762  0.        ]


#### Resultado

**$$ \vec{F} = 67.027 \vec{i} [N] $$**